code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -82,18 +101,20 @@ This file's folder contains:
 			R.emagged = 1
 			R << "<span class='warning'><b>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</b></span>"
 		S.laws = new/datum/ai_laws/ratvar
+		S.laws.associate(S)
```

Comment:
Very important; means you can't simply reset the AI or borg to remove the law, which would accordingly be very bad, as they'd still be an antag and it'd get confusing.